sbc_compassion/models/import_letters_history.py

```diff
@@ -426,7 +434,16 @@ def _find_template(self, img):
         """
         templates = self.env['sponsorship.correspondence.template'].search(
             [('pattern_image', '!=', False)])
-        return pr.find_template(img, templates)
+        threshold = float(
+            self.env.ref('sbc_compassion.threshold_keypoints_template').value)
+        threshold = 0.29
```

Comment:
Thanks I was trying to figure out why my modifications did not work ^^'
It was during the test (faster this way than in xml or gui)